### PR TITLE
[FEATURE] Auto-registration of NewContentElementWizard via TCA

### PR DESCRIPTION
**Pull request by linawolf**
References https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/801

Releases: main

### CODE REVIEW

File: Documentation/PageTsconfig/Mod/Wizards.rst [hunk 1]
```diff
@@ -19,6 +19,13 @@ The `wizards` section allows to customize the *New record wizard* and the
 newContentElement.wizardItems
 =============================
 
+..  versionchanged:: 13.0
+    New content elements added via TCA to the
+    :ref:`items <t3tca:columns-select-properties-items>` of field :sql:`CType`
+    of table :sql:`tt_content` are automatically added to the newContentElement
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/Wizards.rst:25]:
```suggestion
    of table :sql:`tt_content` are automatically added to the New Content Element
```

File: Documentation/PageTsconfig/Mod/Wizards.rst [hunk 2]
```diff
@@ -19,6 +19,13 @@ The `wizards` section allows to customize the *New record wizard* and the
...
+    wizard. The following page TSconfig can be used to override values set via
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/Wizards.rst:26]:
```suggestion
    Wizard. The following page TSconfig can be used to override values set via
```

File: Documentation/PageTsconfig/Mod/Wizards.rst [hunk 3]
```diff
@@ -29,9 +36,26 @@ newContentElement.wizardItems
 
         In the new content element wizard, content element types are grouped
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/Wizards.rst:37]:
```suggestion
        In the New Content Element Wizard, content element types are grouped
```

File: Documentation/PageTsconfig/Mod/Wizards.rst [hunk 4]
```diff
@@ -29,9 +36,26 @@ newContentElement.wizardItems
...
+
+        ..  confval:: removeItems
+            :name: mod-wizards-newContentElement-wizardItems-removeItems
+            :type: comma separated list of groups
+            :Path: mod.wizards.newContentElement.wizardItems.removeItems
+
+            ..  versionchanged:: 13.0
+
+            With this setting one or several groups with all their content
+            elements can be hidden in the "create new content element" wizard.
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/Wizards.rst:53]:
```suggestion
            elements can be hidden in the New Content Element Wizard.
```

File: Documentation/PageTsconfig/Mod/Wizards.rst [hunk 5]
```diff
@@ -59,19 +83,20 @@ newContentElement.wizardItems
...
-                mod.wizards.newContentElement.wizardItems.common.show = text,textpic
+                The previously used option to show / hide elements
+                :typoscript:`mod.wizards.newContentElement.wizardItems.<group>.show` is
+                not evaluated anymore.
 
+                All configured groups and elements are automatically shown. Removing these
+                groups and elements from the New Content Element Wizard can be done via
+                the option :confval:`removeItems <mod-wizards-newContentElement-wizardItems-removeItems>` and
+                :confval:`[group].removeItems <mod-wizards-newContentElement-wizardItems-group-removeItems>`
+                options.
```
Comment (froemken) [inline Documentation/PageTsconfig/Mod/Wizards.rst:99]: Duplicate? I'm unsure

**Review by froemken — CHANGES_REQUESTED** [review]

File: Documentation/PageTsconfig/Mod/Wizards.rst (hunk 5 above)
Comment (linawolf) [reply Documentation/PageTsconfig/Mod/Wizards.rst:99]: Nope, one is for the group, the other is in general, two different options

**Review by froemken — APPROVED** [review]

**Pull request merged**